doc/modules/model_evaluation.rst

```diff
@@ -1959,6 +1959,37 @@ function::
   >>> mean_squared_log_error(y_true, y_pred)
   0.044...
 
+.. _mean_absolute_percentage_error:
+
+Mean absolute percentage error
+------------------------------
+The :func:`mean_absolute_percentage_error` (MAPE), also known as mean absolute
+percentage deviation (MAPD), is an evaluation metric for regression problems.
+The idea of this metric is to be sensitive to relative errors. It is for example
+not changed by a global scaling of the target variable.
+
+If :math:`\hat{y}_i` is the predicted value of the :math:`i`-th sample
+and :math:`y_i` is the corresponding true value, then the mean absolute percentage
+error (MAPE) estimated over :math:`n_{\text{samples}}` is defined as
+
+.. math::
+
+  \text{MAPE}(y, \hat{y}) = \frac{\mid y - \hat{y} \mid}{\text{MAX}(\epsilon, \mid y \mid)}
+
+where :math:`\epsilon` is a very small number greater than zero to avoid
+the crash when y is zero.
+
+The :func:`mean_absolute_percentage_error` function supports multioutput.
+
+Here is a small example of usage of the :func:`mean_absolute_percentage_error`
+function::
+
+  >>> from sklearn.metrics import mean_absolute_percentage_error
+  >>> y_true = [3, -0.5, 2, 7]
```

Comment:
I would rather not put negative value in the canonical doctest for this scoring metric. While MAPE is technically defined in this case, MAPE is really not intended to be used with smalle `y_true` values that oscillate around 0.

I would rather highlight is behavior with `y_true` values that take widely scaled positive values, e.g. `[1., 10, 1e6]` and that approximately follows those scales `y_pred = [0.9, 15, 1.2e6]`.

Reply:
done